library/std/Cargo.toml

```diff
@@ -42,7 +42,7 @@ dlmalloc = { version = "0.2.3", features = ['rustc-dep-of-std'] }
 fortanix-sgx-abi = { version = "0.3.2", features = ['rustc-dep-of-std'] }
 
 [target.'cfg(target_os = "hermit")'.dependencies]
-hermit-abi = { version = "0.2.0", features = ['rustc-dep-of-std'] }
+hermit-abi = { version = "0.2.2", features = ['rustc-dep-of-std'] }
```

Comment:
Please commit an updated `Cargo.lock` which reflects the bump to 0.2.2.

Reply:
Yes, I forgot to add the file